### PR TITLE
Fix link name in deployment guide

### PR DESCRIPTION
**Pull request by kota2and3kan**
This PR updates the link (document) name in the manual deployment guide based on the https://github.com/scalar-labs/scalar-kubernetes/pull/200#discussion_r1268854512 in another PR.

Please take a look!

### CODE REVIEW

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

File: docs/ManualDeploymentGuideScalarDBServerOnAKS.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ You must get the ScalarDB Server container image by visiting [Azure Marketplace]
 
 ## Step 2. Create an AKS cluster
 
-You must create an AKS cluster for the ScalarDB Server deployment. For details, see [Create an AKS cluster for Scalar products](./CreateAKSClusterForScalarProducts.md).
+You must create an AKS cluster for the ScalarDB Server deployment. For details, see [A guideline for creating an AKS cluster for Scalar products](./CreateAKSClusterForScalarProducts.md).
```
Comment (josh-wong) [inline docs/ManualDeploymentGuideScalarDBServerOnAKS.md:21]:
```suggestion
You must create an AKS cluster for the ScalarDB Server deployment. For details, see [Guidelines for creating an AKS cluster for Scalar products](./CreateAKSClusterForScalarProducts.md).
```

File: docs/ManualDeploymentGuideScalarDBServerOnEKS.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ You must get the ScalarDB Server container image by visiting [AWS Marketplace](h
 
 ## Step 2. Create an EKS cluster
 
-You must create an EKS cluster for the ScalarDB Server deployment. For details, see [Create an Amazon EKS cluster for Scalar products](./CreateEKSClusterForScalarProducts.md).
+You must create an EKS cluster for the ScalarDB Server deployment. For details, see [A guideline for creating an Amazon EKS cluster for Scalar products](./CreateEKSClusterForScalarProducts.md).
```
Comment (josh-wong) [inline docs/ManualDeploymentGuideScalarDBServerOnEKS.md:21]:
```suggestion
You must create an EKS cluster for the ScalarDB Server deployment. For details, see [Guidelines for creating an Amazon EKS cluster for Scalar products](./CreateEKSClusterForScalarProducts.md).
```

File: docs/ManualDeploymentGuideScalarDLAuditorOnAKS.md [hunk 3]
```diff
@@ -24,11 +24,11 @@ You must get the ScalarDL Ledger and ScalarDL Auditor container images by visiti
 
 ## Step 2. Create an AKS cluster for ScalarDL Ledger
 
-You must create an AKS cluster for the ScalarDL Ledger deployment. For details, see [Create an AKS cluster for Scalar products](./CreateAKSClusterForScalarProducts.md).
+You must create an AKS cluster for the ScalarDL Ledger deployment. For details, see [A guideline for creating an AKS cluster for Scalar products](./CreateAKSClusterForScalarProducts.md).
```
Comment (josh-wong) [inline docs/ManualDeploymentGuideScalarDLAuditorOnAKS.md:27]:
```suggestion
You must create an AKS cluster for the ScalarDL Ledger deployment. For details, see [Guidelines for creating an AKS cluster for Scalar products](./CreateAKSClusterForScalarProducts.md).
```

File: docs/ManualDeploymentGuideScalarDLAuditorOnAKS.md [hunk 4]
```diff
@@ -24,11 +24,11 @@ You must get the ScalarDL Ledger and ScalarDL Auditor container images by visiti
...
 
 ## Step 3. Create an AKS cluster for ScalarDL Auditor
 
-You must also create an AKS cluster for the ScalarDL Auditor deployment. For details, see [Create an AKS cluster for Scalar products](./CreateAKSClusterForScalarProducts.md).
+You must also create an AKS cluster for the ScalarDL Auditor deployment. For details, see [A guideline for creating an AKS cluster for Scalar products](./CreateAKSClusterForScalarProducts.md).
```
Comment (josh-wong) [inline docs/ManualDeploymentGuideScalarDLAuditorOnAKS.md:31]:
```suggestion
You must also create an AKS cluster for the ScalarDL Auditor deployment. For details, see [Guidelines for creating an AKS cluster for Scalar products](./CreateAKSClusterForScalarProducts.md).
```

File: docs/ManualDeploymentGuideScalarDLAuditorOnEKS.md [hunk 5]
```diff
@@ -24,11 +24,11 @@ You must get the ScalarDL Ledger and ScalarDL Auditor container images from [AWS
 
 ## Step 2. Create an EKS cluster for ScalarDL Ledger
 
-You must create an EKS cluster for the ScalarDL Ledger deployment. For details, see [Create an Amazon EKS cluster for Scalar products](./CreateEKSClusterForScalarProducts.md).
+You must create an EKS cluster for the ScalarDL Ledger deployment. For details, see [A guideline for creating an Amazon EKS cluster for Scalar products](./CreateEKSClusterForScalarProducts.md).
```
Comment (josh-wong) [inline docs/ManualDeploymentGuideScalarDLAuditorOnEKS.md:27]:
```suggestion
You must create an EKS cluster for the ScalarDL Ledger deployment. For details, see [Guidelines for creating an Amazon EKS cluster for Scalar products](./CreateEKSClusterForScalarProducts.md).
```

File: docs/ManualDeploymentGuideScalarDLAuditorOnEKS.md [hunk 6]
```diff
@@ -24,11 +24,11 @@ You must get the ScalarDL Ledger and ScalarDL Auditor container images from [AWS
...
 
 ## Step 3. Create an EKS cluster for ScalarDL Auditor
 
-You must also create an EKS cluster for the ScalarDL Auditor deployment. For details, see [Create an Amazon EKS cluster for Scalar products](./CreateEKSClusterForScalarProducts.md).
+You must also create an EKS cluster for the ScalarDL Auditor deployment. For details, see [A guideline for creating an Amazon EKS cluster for Scalar products](./CreateEKSClusterForScalarProducts.md).
```
Comment (josh-wong) [inline docs/ManualDeploymentGuideScalarDLAuditorOnEKS.md:31]:
```suggestion
You must also create an EKS cluster for the ScalarDL Auditor deployment. For details, see [Guidelines for creating an Amazon EKS cluster for Scalar products](./CreateEKSClusterForScalarProducts.md).
```

File: docs/ManualDeploymentGuideScalarDLOnAKS.md [hunk 7]
```diff
@@ -12,7 +12,7 @@ You must get the ScalarDL Ledger container image from [Azure Marketplace](https:
 
 ## Step 2. Create an AKS cluster
 
-You must create an AKS cluster for the ScalarDL Ledger deployment. For details, see [Create an AKS cluster for Scalar products](./CreateAKSClusterForScalarProducts.md).
+You must create an AKS cluster for the ScalarDL Ledger deployment. For details, see [A guideline for creating an AKS cluster for Scalar products](./CreateAKSClusterForScalarProducts.md).
```
Comment (josh-wong) [inline docs/ManualDeploymentGuideScalarDLOnAKS.md:15]:
```suggestion
You must create an AKS cluster for the ScalarDL Ledger deployment. For details, see [Guidelines for creating an AKS cluster for Scalar products](./CreateAKSClusterForScalarProducts.md).
```

File: docs/ManualDeploymentGuideScalarDLOnEKS.md [hunk 8]
```diff
@@ -12,7 +12,7 @@ You must get the ScalarDL Ledger container image from [AWS Marketplace](https://
 
 ## Step 2. Create an EKS cluster
 
-You must create an EKS cluster for the ScalarDL Ledger deployment. For details, see [Create an Amazon EKS cluster for Scalar products](./CreateEKSClusterForScalarProducts.md).
+You must create an EKS cluster for the ScalarDL Ledger deployment. For details, see [A guideline for creating an Amazon EKS cluster for Scalar products](./CreateEKSClusterForScalarProducts.md).
```
Comment (josh-wong) [inline docs/ManualDeploymentGuideScalarDLOnEKS.md:15]:
```suggestion
You must create an EKS cluster for the ScalarDL Ledger deployment. For details, see [Guidelines for creating an Amazon EKS cluster for Scalar products](./CreateEKSClusterForScalarProducts.md).
```

**Review by josh-wong — CHANGES_REQUESTED** [review]
I've made a slight change to the doc titles and updated the link names. PTAL!

**Comment by kota2and3kan** [comment]
@josh-wong  
Thank you for your review and suggestions!
I applied your suggestions. Please take a look when you have time!

**Pull request merged**